e2e/cypress/support/api/system.js

```diff
@@ -123,8 +123,8 @@ export const getDefaultConfig = () => {
 
     if (isCloud) {
         fromCypressEnv.CloudSettings = {
-            CWSUrl: cypressEnv.cwsUrl,
-            CWSAPIUrl: cypressEnv.cwsApiUrl,
+            CWSURL: cypressEnv.cwsUrl,
```

Comment:
Opposite my question on `e2e/cypress.json`, if that was intentional, should `cwsUrl` be renamed then?

Reply:
Good catch. Fixed in edb0b6fc1.